apps/studio/components/grid/components/grid/Grid.tsx

```diff
@@ -214,7 +219,20 @@ export const Grid = memo(
                               </p>
                               <div className="flex items-center space-x-2 mt-4">
                                 {onAddRow !== undefined && onImportData !== undefined && (
-                                  <Button type="default" onClick={onImportData}>
+                                  <Button
+                                    type="default"
+                                    onClick={() => {
+                                      onImportData()
+                                      sendEvent({
+                                        action: TelemetryActions.IMPORT_DATA_BUTTON_CLICKED,
```

Comment:
You want to add this one to this button too:

<img width="411" alt="Screenshot 2025-01-28 at 21 38 00" src="https://github.com/user-attachments/assets/e68ab7e0-1ba5-4bc2-9538-e6c4a24b38fd" />